internal/mode/static/state/change_processor.go

```diff
@@ -176,7 +171,7 @@ func NewChangeProcessorImpl(cfg ChangeProcessorConfig) *ChangeProcessorImpl {
 			}
```

Comment:
Can you also update the references to 0.7.1 on lines 163-166?

Reply:
I was going to, but it said "as of", which made it sound like it was introduced in that version. Unless we mean "as of the latest version"

Reply:
We mean "as of the latest version". Feel free to reword though cause it is a bit misleading